tools/meson-hwdb-update.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -ex
+
+cd "$1"
+wget -O usb.ids 'http://www.linux-usb.org/usb.ids'
```

Comment:
It's a straightforward conversion of the existing make rule. That said, it makes sense to use curl instead, because libcurl is a build dependency.